trin-core/src/cli.rs

```diff
@@ -118,16 +118,22 @@ pub struct TrinConfig {
 
     #[structopt(
         long = "trusted-provider",
-        help = "Trusted provider to use. (options: 'infura' (default) or 'geth' (experimental))",
+        help = "Trusted provider to use. (options: 'infura' (default), 'pandaops' (devops) or 'local' (experimental))",
         default_value(DEFAULT_TRUSTED_PROVIDER)
     )]
     pub trusted_provider: TrustedProviderType,
 
     #[structopt(
-        long = "geth-url",
-        help = "URL for geth client to use as trusted http provider (experimental)"
+        long = "pandaops-url",
+        help = "URL for pandaops client to use as trusted http provider (experimental)"
     )]
-    pub geth_url: Option<String>,
+    pub pandaops_url: Option<String>,
+
+    #[structopt(
+        long = "local-provider-url",
+        help = "URL for local Ethereum execution client as '<ip_address>:<port>' (experimental)."
+    )]
+    pub local_provider_url: Option<String>,
```

Comment:
Any reason not to combine `pandaops-url` and `local-provider-url` into a `trusted-provider-url`? I think it would help clean up some of the following parsing logic. eg.
- if `infura` -> no `trusted-provider-url` is allowed
- else if `pandaops` -> validate `trusted-provider-url` is a valid url (using the `url` library)
- else if `local` -> validate `trusted-provider-url` is valid and perform the port collision check

Reply:
This makes sense. Not done initially to minimise disrupting the infura/pandaops logic

Reply:
Done. I have changed all instances of the word "local" to "custom", because one is not limited to using a local node with this flag (it could be also be personal remote node)